.github/workflows/production-suite.yml

```diff
@@ -203,3 +203,57 @@ jobs:
         env:
           ZULIP_BOT_KEY: ${{ secrets.ZULIP_BOT_KEY }}
         run: /tmp/send-failure-message
+
+  production_upgrade:
```

Comment:
It's probably worth a block comment here explaining that this test suite is meant to verify that upgrading from a previous major release to master works.

Reply:
Sure.